service/src/test/java/uk/gov/hmcts/reform/fpl/service/orders/generator/OrderDocumentGeneratorHolderTest.java

```diff
@@ -136,4 +148,27 @@ void typeToAdditionalDocsCollectorCached() {
 
         collectors.forEach(collector -> verify(collector).accept());
     }
+
+    @Test
+    void shouldReturnAppropriateNotificationDocumentGenerators() {
+        Map<Order, DocmosisParameterGenerator> expectedNotificationDocumentPerType = Map.of(
+            A70_PLACEMENT_ORDER, a206PlacementOrderNotificationParameterGenerator
+        );
+
+        //Check expected orders match expected generators
```

Comment:
there is one entry in a map, no need for looping. Anyway doing assertion is a loop is usually bad idea, if you have 20 elements and assertion fails on 10th element you will get info that line number X failed assertion, but you wont know what iteration had problem.

Reply:
Good point. However, I wanted to make this solution scalable if we ever have more items in this map.
Having said that, I'll add a description to each assertion so that, if it fails, it will tell us exactly which order has failed.

Hope that's ok with you.